### PR TITLE
dpp 0.4.9 (new formula)

### PR DESCRIPTION
**Pull request by FnControlOption**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/dpp.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
+class Dpp < Formula
+  desc "Directly include C headers in D source code"
+  homepage "https://github.com/atilaneves/dpp"
+  url "https://github.com/atilaneves/dpp.git",
+    tag:      "v0.4.9",
+    revision: "55daf91e2c180962de783752b3e4447850b7a151"
+  license "BSL-1.0"
+
+  pour_bottle? only_if: :clt_installed
```
Comment (chenrui333) [inline Formula/dpp.rb:9]: maybe add a comment for this line?

Comment (FnControlOption) [reply Formula/dpp.rb:9]: What should the comment say?

Comment (chenrui333) [reply Formula/dpp.rb:9]: Something like what is in `gcc` and `llvm` formulae, mostly just specify the reason why this is needed.

Comment (FnControlOption) [reply Formula/dpp.rb]: Pushed a commit to add both the CLT and Xcode toolchains to rpath, so this shouldn't be needed anymore.

File: Formula/dpp.rb [hunk 2]
```diff
@@ -0,0 +1,55 @@
+class Dpp < Formula
+  desc "Directly include C headers in D source code"
+  homepage "https://github.com/atilaneves/dpp"
+  url "https://github.com/atilaneves/dpp.git",
+    tag:      "v0.4.9",
+    revision: "55daf91e2c180962de783752b3e4447850b7a151"
```
Comment (carlocab) [inline Formula/dpp.rb:6]:
```suggestion
      tag:      "v0.4.9",
      revision: "55daf91e2c180962de783752b3e4447850b7a151"
```

**Review by chenrui333 — APPROVED** [review]
lgtm

**Pull request closed without merging**